todo_app/data/models.py

```diff
@@ -0,0 +1,26 @@
+# app/models.py
+
+from flask_sqlalchemy import SQLAlchemy
+from flask_login import UserMixin, LoginManager
+from flask_dance.consumer.storage.sqla import OAuthConsumerMixin
+
+
+db = SQLAlchemy()
```

Comment:
You're using a SQL library despite us using a MongoDB non-relational database; by default it looks like it's just falling back to an in-memory SQLite DB which somewhat defeats the purpose of a database since it won't be shared between different instances of our app; loading the info via session would likely make more sense (since the info is then actually stored (signed) client side, and can be shared between servers)